### PR TITLE
feature/nam_rap_precip_snowfall_stats

### PR DESCRIPTION
**Pull request by malloryprow**
## Pull Request Testing ##

- Add precipitation and snowfall stats for NAM and RAP

- [ ] Describe testing already performed for this Pull Request:</br>

> Jobs have been running in a cron for a few weeks now

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- Check out branch
- Copy /lfs/h2/emc/vpppg/noscrub/mallory.row/evs_regional_atmos_test_scripts/pr_116/crontab.nam.rap.precip.snowfall
- Run sed -i "s/{USER}/$USER/g" on your copy of crontab.nam.rap.precip.snowfall
- Set the crons in your crontab

- [ ] Has the code been checked to ensure that no errors occur during the execution? **Yes**

- [ ] Do these updates/additions include sufficient testing updates? **Yes**

- [ ] Please complete this pull request review by **06/01/2023**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
Hi @malloryprow , I will pick up this PR for you.

**Comment by ShelleyMelchior-NOAA** [comment]
I modified the 4 ecf scripts to run from my installation ($HOMEevs). Please review the output stats files.
output: `/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale`

script: `jevs_nam_precip_stats.ecf`
log: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_nam_precip_stats_00.o58269832`

script: `jevs_nam_snowfall_stats.ecf`
log: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_nam_snowfall_stats_00.o58271870`

script: `jevs_rap_precip_stats.ecf`
log: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_rap_precip_stats_00.o58271874`

script: `jevs_rap_snowfall_stats.ecf`
log: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_rap_snowfall_stats_00.o58271892`

**Comment by malloryprow** [comment]
Thanks! I see the output in /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/mesoscale. Would it be possible t get the crons set up and run over the weekend? I want to make sure we get the final .stat file.

**Comment by ShelleyMelchior-NOAA** [comment]
Yep! Glad the one time runs look ok. I will follow the cron instructions noted at the top of the PR and let you know when they are up and running.

**Comment by malloryprow** [comment]
Thank you!! :) I'm taking a half day today, so I will check Monday. That will give us a good few days to compare with my run.

**Comment by malloryprow** [comment]
✅ Checked the files in nam.20230519 and rap.20230519 (stats from 20230520 are running today). All looks good and matches my files!

File: ecf/mesoscale/stats/jevs_nam_precip_stats.ecf
Comment (ShelleyMelchior-NOAA) [inline ecf/mesoscale/stats/jevs_nam_precip_stats.ecf:1]: Do we need to state shell interpreter at top of ecf scripts?

Comment (malloryprow) [reply ecf/mesoscale/stats/jevs_nam_precip_stats.ecf:1]: It is set with "#PBS -S /bin/bash"

File: jobs/mesoscale/stats/JEVS_MESOSCALE_STATS
Comment (ShelleyMelchior-NOAA) [inline jobs/mesoscale/stats/JEVS_MESOSCALE_STATS:1]: The changes for JEVS_MESOSCALE_STATS look good. Just need to make sure @RoshanShrestha-NOAA is aware of these modifications to make sure he can integrate them into his feature branch in his fork.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Talked to Roshan and he knows to integrate JEVS_MESOSCALE_STATS in to his feature branch of his fork.

**Pull request merged**